cpp/src/io/orc/orc_gpu.hpp

```diff
@@ -29,6 +29,7 @@
 #include <cudf/types.hpp>
 #include <cudf/utilities/span.hpp>
 
+#include <cuco/static_map.cuh>
```

Comment:
Theoratically, cuco is a different header group from rmm and is considered "further" so this should be placed after `#include <rmm/cuda_stream_view.hpp>` with a blank line in between.

Reply:
sorted :+1: